wherehows-backend/app/models/daos/EtlJobDao.java

```diff
@@ -13,63 +13,57 @@
  */
 package models.daos;
 
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import java.util.Date;
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
-import metadata.etl.models.EtlJobName;
 import metadata.etl.models.EtlJobStatus;
 import org.springframework.dao.DataAccessException;
 import org.springframework.jdbc.support.KeyHolder;
 import play.libs.Time;
 import utils.JdbcUtil;
-import utils.JsonUtil;
-
-import java.sql.SQLException;
 
 
 /**
  * Created by zechen on 9/25/15.
  */
 public class EtlJobDao {
-  public static final String GET_ALL_SCHEDULED_JOBS = "SELECT * FROM wh_etl_job_schedule";
+
+  public static final String GET_ALL_SCHEDULED_JOBS =
+      "SELECT wh_etl_job_name, enabled, next_run FROM wh_etl_job_schedule";
 
   public static final String UPDATE_NEXT_RUN =
-    "REPLACE INTO wh_etl_job_schedule(wh_etl_job_name, next_run) VALUES (:whEtlJobName, :nextRun)";
+      "UPDATE wh_etl_job_schedule SET next_run = :nextRun WHERE wh_etl_job_name = :whEtlJobName";
```

Comment:
I think we need to keep REPLACE INTO here so we don't need to pre-populate the table for every new ETL job.